packages/forms-web-app/src/lib/client-side/comment-actions.js

```diff
@@ -0,0 +1,38 @@
+document.addEventListener('DOMContentLoaded', () => {
+	const readMoreButtons = document.querySelectorAll('.read-more');
+	const closeButtons = document.querySelectorAll('.close-button');
+	readMoreButtons.forEach((button) => {
```

Comment:
It might not be worth doing but it would be preferable to make this reusable and work with a njk macro

In my head that would look something like:
```
attach on click event to page body
  check if target has .read-more class
  lookup text via a parent element by classname e.g. read-more-container
  toggle text content
  toggle button text content
```